src/test/python/MCSVerification.py

```diff
@@ -568,14 +679,26 @@ def main(argv):
     ssDir = workingDir + '/scisparkGTG'
 
     try:
-        opts, _ = getopt.getopt(argv, "h")
-        if len(opts) == 1:
-            if '-h' in opts[0][0]:
+        opts, arg = getopt.getopt(argv, "hs:")
+
+        if len(opts) == 0:
+            print 'Please run: python MCSVerification.py -h'
+            sys.exit(2)
+
+        for opt, arg in opts:
+            if '-h' in opt: 
                 print 'These are the verification test to determine if SciSpark GTG is operating correctly.'
+                print 'To run: python MCSVerification.py -s scalaVersion'
+                print 'For example: python MCSVerification.py -s scala-2.11'
                 print 'The data used in the runs were used in the paper, and the python GTG results are downloaded.'
                 print 'If needed, the original GTG can de downloaded and used to generate the results. \
                     Please use the original release - Release 1.0'
                 sys.exit(2)
+            elif '-s' in opt: 
+                scalaVersion = arg
+                if not os.path.exists('../../../target/' + scalaVersion):
```

Comment:
Ack'ed @sujen1412. I really do not anticipate the user will run the verification test from other location other than the test folder, hence it might not be worth the effort. However, how do you suggests we get rid of the relative path.